client/state/data-layer/wpcom/me/notification/settings/test/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { NOTIFICATION_SETTINGS_UPDATE, NOTICE_CREATE } from 'state/action-types';
+import {
+	requestNotificationSettings,
+	updateSettings,
+	handleError,
+} from '../';
+
+describe( 'wpcom-api', () => {
```

Comment:
Do we need all of these describes? Currently test output looks like
<img width="360" alt="screen shot 2017-06-08 at 1 55 44 pm" src="https://user-images.githubusercontent.com/1270189/26950405-54515e06-4c52-11e7-8af6-e3588029a301.png">